dev/how-to/maintain/identify-slow-queries.md

```diff
@@ -43,19 +46,18 @@ select * from t_slim, t_wide where t_slim.c0=t_wide.c0;
 * `Txn_start_ts`: The start timestamp of the transaction (transaction ID). You can use this value to `grep` the transaction-related logs.
 * `User`: The name of the user who executes this statement.
 * `Conn_ID`: The Connection ID (session ID). For example, you can use the keyword `con:3` to `grep` the log whose session ID is 3.
-* `Query_time`: Indicates the execution time of this statement. Only the statement whose execution time exceeds slow-threshold outputs this log (the unit is second). The unit of all the following time fields is second.
+* `DB`: The current database.
+* `Index_ids`: The IDs of the indexes involved in the statement.
+* `Is_internal`: Whether the SQL statement is TiDB internal. `true` indicates that the SQL statement is executed internally in TiDB, such as `analyze`, `load variables`, etc. `false` indicates the SQL statement is executed by the user.
```

Comment:
```suggestion
* `Is_internal`: Whether the SQL statement is TiDB internal. `true` indicates that the SQL statement is executed internally in TiDB, such as `analyze`, `load variables`, etc. `false` indicates that the SQL statement is executed by the user.
```

Reply:
Your expression is right. But in technical writing, it is recommended to add "that" here.